python/tests/test_petab.py

```diff
@@ -0,0 +1,60 @@
+from petab.C import *
+import parpe.petab
+import pandas as pd
+
+
+def test_get_parameter_override_id_to_placeholder_id():
+    """Test get_parameter_override_id_to_placeholder_id"""
+
+    observable_df = pd.DataFrame(data={
+        OBSERVABLE_ID: ['obs1', 'obs2'],
+        OBSERVABLE_FORMULA: ['1.0',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['OBSERVABLE_FORMULA' may be undefined, or defined from star imports: petab.C](https://app.codacy.com/manual/dweindl/parPE/pullRequest?prid=4922781)